catalyst-gateway/bin/src/cli.rs

```diff
@@ -46,6 +52,12 @@ impl Cli {
                 service::run(&settings.address, state).await?;
                 Ok(())
             },
+            Self::Docs(settings) => {
+                let docs = service::get_app_docs(&settings.format);
+                let mut docs_file = std::fs::File::create(settings.output)?;
```

Comment:
The ticket specifically says that we need the spec to print to stdout unless an optional `--output` parameter is given.
We also do not want or need a format option.  Only produce pretty printed json.

Reply:
Sorry, missed that :) updating